src/data-search.js

```diff
@@ -45,6 +46,15 @@ export async function getData () {
   };
   const collection = await Promise.all(config.map(doSearches));
 
+  // Find and save final link for each paper's DOI
+  const { papers } = collection;
+  for (const paper in papers) {
```

Comment:
I attempted using the JS destructuring method here but since papers is an array within the collection object it doesn't seem to work the same way as it does with keywords in line 41. Any better ways to carry out the same function? I've been stuck with this for some time since destructuring is also new to me.